src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ECEAssistantRequirements.vue

```diff
@@ -42,9 +42,3 @@ export default defineComponent({
   name: "ECEAssistantRequirements",
 });
 </script>
-
-<style>
-.title-header {
-  line-height: 1.25;
-}
-</style>
```

Comment:
could we check that this still works on mobile? cause I remember without this piece when text would get all messed up when we viewed it in mobile mode.

Reply:
I just checked. It seems like it will still work ... interesting, we don't need this.